util/precompress/precompress.go

```diff
@@ -63,13 +64,19 @@ type Options struct {
 // OpenPrecompressedFile opens a file from fs, preferring compressed versions
 // generated by PrecompressDir if possible.
 func OpenPrecompressedFile(w http.ResponseWriter, r *http.Request, path string, fs fs.FS) (fs.File, error) {
-	if tsweb.AcceptsEncoding(r, "br") {
+	if tswebutil.AcceptsEncoding(r, "zstd") {
+		if f, err := fs.Open(path + ".zst"); err == nil {
+			w.Header().Set("Content-Encoding", "zstd")
+			return f, nil
+		}
+	}
+	if tswebutil.AcceptsEncoding(r, "br") {
```

Comment:
Maybe it's time to drop Brotli to make up for the increased cost/size of adding zstd?

Safari users can use gzip for now?